src/components/ProjectDescComp.js

```diff
@@ -90,7 +100,7 @@ const YumYum = () => {
 
   return (
     <LeftLightGreenBlock>
-      <LeftContainer>
+      <LeftContainer onClick={() => navigate("/projects/yumyum")}>
```

Comment:
why are we using `navigate` here? Cant these just be `<a>` tags?

Reply:
Will get to this